README.md

```diff
@@ -1,3 +1,56 @@
 # Watchdog
 
 A package to provide improved watchdog functionality.
+
+Only works on an actual ESP.
```

Comment:
Maybe say that it doesn't work on hosts? Or use 'ESP32' instead of 'ESP'?

Reply:
Not mentioning host, but improved it.